client/ayon_core/modules/deadline/deadline_module.py

```diff
@@ -19,23 +19,23 @@ class DeadlineModule(AYONAddon, IPluginPaths):
 
     def initialize(self, studio_settings):
         # This module is always enabled
-        deadline_urls = {}
+        deadline_server_info = {}
```

Comment:
Should this be `deadline_servers_info` since it's actually info per server? (so multiple servers?)